ds/beis_indicators/nomis/ni_processing.py

```diff
@@ -0,0 +1,77 @@
+import pandas as pd
+import beis_indicators
+
+PROJECT_DIR = beis_indicators.project_dir
+NI_BRES_2019_PUB_TABLES = "https://www.nisra.gov.uk/sites/nisra.gov.uk/files/publications/BRES2019-Publication-Tables.xlsx"
+
+
+def sic4_to_cluster_lookup(
+    sic4_lookup_csv=f"{PROJECT_DIR}/data/raw/sic_4_industry_segment_lookup.csv",
+):
+    """Use SIC4 to cluster namelookup csv file to create a dictionary
```

Comment:
What are the clusters that are being referred to?

Reply:
Clusters are the clusters in `sic_4_industry_segment_lookup.csv` for example `manufacture_aerospace`